File: kotlin/1.A.kt
fun main(){
    val (n,m,a)=readLine()!!.split(" ").map(String::toLong)
    val r= if (n%a==0L) n/a else n/a+1
    val k= if (m%a==0L) m/a else m/a+1
    print(r*k)
}

File: dotNet/1.A.cs
using System;
namespace cf
{
    public class A1
    {
        public static void Main(String[] args)
        {
            string[] s = Console.ReadLine().Split(' ');
            Int64 n = int.Parse(s[0]);
            Int64 m = int.Parse(s[1]);
            Int64 a=int.Parse(s[2]);
            Int64 r=n/a;
            Int64 k = m / a;
            if (n % a != 0)
            {
                r += 1;
            }
            if (m % a != 0) k += 1;

            Console.WriteLine(r*k);
            

        }
    }
}

